examples/core/core_input_mouse_wheel.c

```diff
@@ -31,7 +33,7 @@ int main(void)
     {
         // Update
         //----------------------------------------------------------------------------------
-        boxPositionY -= (GetMouseWheelMove()*scrollSpeed);
+        boxPosition = Vector2Subtract(boxPosition, Vector2Scale(GetMouseWheelMoveV(), scrollSpeed));
```

Comment:
Please, could we keep this file as it was, for most novice users `GetMouseWheelMove()` is enough, no need to increase in one level of complexity this basic example.